internal/ethapi/api.go

```diff
@@ -1807,9 +1892,28 @@ func (api *TransactionAPI) GetRawTransactionByHash(ctx context.Context, hash com
 // GetTransactionReceipt returns the transaction receipt for the given transaction hash.
 func (api *TransactionAPI) GetTransactionReceipt(ctx context.Context, hash common.Hash) (map[string]interface{}, error) {
 	found, tx, blockHash, blockNumber, index, err := api.b.GetTransaction(ctx, hash)
-	if err != nil {
-		return nil, NewTxIndexingError() // transaction is not fully indexed
+	if api.b.GetPreconfirmationForwardingURL() == "" {
```

Comment:
Should it be obtained from the forward URL first? If appeared error then get data from `GetTransaction`.